data_validation/__main__.py

```diff
@@ -245,16 +245,13 @@ def build_config_managers_from_args(args):
     )
 
     is_filesystem = source_client._source_type == "FileSystem"
-    tables_list = cli_tools.get_tables_list(
-        args.tables_list, default_value=[], is_filesystem=is_filesystem
-    )
 
-    for table_obj in tables_list:
+    if args.tables_list is None and validate_cmd == "Custom-query":
```

Comment:
```suggestion
    if args.tables_list is None and config_type == consts.CUSTOM_QUERY:
```
It'd be nice to reference the constant here and use the config type variable as it is used on the `config_type != consts.SCHEMA_VALIDATION`